config_denoising.yaml

```diff
@@ -0,0 +1,23 @@
+dataset:
```

Comment:
Do you think we should provide somewhere in the repo example/template configs? or perhaps even in examples, each example involving training can be used in either notebook mode or using the provided config that we provide?